eth/chains/base.py

```diff
@@ -524,7 +541,10 @@ def get_canonical_block_hash(self, block_number: BlockNumber) -> Hash32:
         """
         return self.chaindb.get_canonical_block_hash(block_number)
 
-    def build_block_with_transactions(self, transactions, parent_header=None):
+    def build_block_with_transactions(self,
```

Comment:
same here, preferred way is to move the self one line down to get more horizontal space